docs/content/ingestion/index.md

```diff
@@ -190,6 +190,34 @@ For example, using Avro Hadoop parser with custom reader's schema file:
 }
 ```
 
+### Parquet Hadoop Parser
+
+This is for batch ingestion using the HadoopDruidIndexer, ike `Avro Hadoop Parser` above. The `inputFormat` of `inputSpec` in `ioConfig` must be set to `"io.druid.data.input.avro.DruidAvroParquetInputFormat"`.
+
```

Comment:
can you change it to following?

Data stored in `parquet` format can be ingested using avro parser by replacing `io.druid.data.input.avro.AvroValueInputFormat` with `io.druid.data.input.avro.DruidAvroParquetInputFormat` in the avro hadoop parser. See an example below.